checkov/terraform/checks/graph_checks/aws/ALBRedirectsHTTPToHTTPS.yaml

```diff
@@ -0,0 +1,70 @@
+metadata:
```

Comment:
can we add a YAML validation test?

Reply:
We already have a test that loads the checks - https://github.com/nimrodkor/checkov/blob/0dc258b532086cd5fb68fe58ee12e1b693c56cc7/tests/graph/terraform/checks/test_yaml_policies.py#L166
If they are malformed, it would fail